x-pack/plugins/observability_solution/slo/server/services/transform_generators/apm_transaction_duration.ts

```diff
@@ -179,7 +179,9 @@ export class ApmTransactionDurationTransformGenerator extends TransformGenerator
               goodEvents: 'slo.numerator.value',
               totalEvents: 'slo.denominator.value',
             },
-            script: `params.goodEvents / params.totalEvents >= ${slo.objective.timesliceTarget} ? 1 : 0`,
+            script: `params.goodEvents / params.totalEvents ${
+              slo.objective.timesliceTarget === 0 ? '>' : '>='
+            } ${slo.objective.timesliceTarget} ? 1 : 0`,
```

Comment:
i will appreciate if you can extract this into a function and reuse

Reply:
I could extract the timesliceTarget comparator part if that's what you referring to, but not the whole script as it is highly coupled to the above bucket_path variables. It would be error prone to have "params.goodEvents" used in one file and defined in a different file.
I'm going to assume it's the target comparator that needs to be extracted